source/common/upstream/conn_pool_map_impl.h

```diff
@@ -60,6 +60,20 @@ ConnPoolMap<KEY_TYPE, POOL_TYPE>::getPool(KEY_TYPE key, const PoolFactory& facto
   return std::ref(*inserted.first->second);
 }
 
+template <typename KEY_TYPE, typename POOL_TYPE>
+bool ConnPoolMap<KEY_TYPE, POOL_TYPE>::erasePool(const KEY_TYPE& key) {
```

Comment:
I don't see this return value used in the prod code. Can you use it even if only asserting it? Otherwise drop it?

Reply:
assert added where it is used in the prod code.